rpi3-convert-stage-5.sh

```diff
@@ -180,6 +181,9 @@ do_install_bootloader() {
 
   detach_device_maps ${sdimgmappings[@]}
 
+  [[ $keep -eq 0 ]] && { rm -rf ${output_dir}/config.txt ${output_dir}/cmdline.txt
```

Comment:
Separate **rm** commands with **;** sign. No need to use **recursive** for removing files.

Reply:
Done